### PR TITLE
09: Page Redesign

### PR DESCRIPTION
**Pull request by gabrielizalo**
# Summay

* Updated html and scss files to change page layout with CSS Grid.

# Trello Card

https://trello.com/c/atVH4WGv/12-redise%C3%B1o-de-la-p%C3%A1gina

# Screenshot

html css training 2018-10-03 14-37-46

### CODE REVIEW

File: maquetado/scss/_styles.scss [hunk 1]
```diff
@@ -8,8 +8,9 @@ body {
 
 // Sections
 .section {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 170px auto auto auto 170px;
```
Comment (henryzarza) [inline maquetado/scss/_styles.scss:12]: Change this for `170px repeat(3, auto) 170px`

File: maquetado/scss/_styles.scss [hunk 2]
```diff
@@ -8,8 +8,9 @@ body {
...
+  grid-template-rows: 100px auto auto;
```
Comment (henryzarza) [inline maquetado/scss/_styles.scss:13]: Change this for `100px repeat(2, auto)`

File: maquetado/scss/_tech.scss [hunk 3]
```diff
@@ -1,31 +1,13 @@
...
   align-items: center;
-  background-color: $green;
-  border-radius: $tech-border-radius;
+  background-color: $white;
   display: flex;
   flex-direction: column;
   margin: 20px;
   max-width: 100%;
   padding: 15px;
   position: relative;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss:9]: Delete this property, isn't necessary

File: maquetado/scss/_tech.scss [hunk 4]
```diff
@@ -48,3 +30,30 @@
   font-size: $text-small;
   text-align: center;
 }
+
+.tech-react, .tech-ruby {
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss:34]: You could do this with a better structure. Remember that tell you 😄

**Review by henryzarza — CHANGES_REQUESTED** [review]

**Comment by gabrielizalo** [comment]
Because better practices using CSS Grid this brach will be discarded.

**Pull request closed without merging**